tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -516,6 +502,10 @@ private void getToolTestResults(Tool tool) {
                                         .asList(date, toolversion.getId(), tag, runner, pipelineNode.getDisplayName(), result, duration,
                                                 logURL);
                                 report.printAndWriteLine(record);
+                                if (result.equals("SUCCESS")) {
+                                    S3Client s3Client = new S3Client();
+                                    s3Client.createObject(toolId, tag, pipelineNode.getDisplayName(), runner, logContent, epochStartTime);
```

Comment:
Can't this throw exceptions, e.g., for a failed network connection, bad/expired credentials, etc.? Does this need to be handled, or if it throws something that's OK.

Reply:
Already does:
```
[main] ERROR io.dockstore.tooltester.helper.ExceptionHandler - Can't handle report
[main] ERROR io.dockstore.tooltester.helper.ExceptionHandler - com.amazonaws.services.s3.model.AmazonS3Exception: The provided token has expired. (Service: Amazon S3; Status Code: 400; Error Code: ExpiredToken; Request ID: 1B4B7D75BC9E62F0), S3 Extended Request ID: l+FG0H1JNx/XDA4lih//w4PeTJgqfyGdIMeDamevOyX/o8BvdPgHZ1Bf8bCdTCnCHgoFpw7JtnE=
```